Broken windows build

Currently, the render library is failing to build on Windows.  The hard-to-read VS2019 errors are below:

25>FlowRenderer.obj : error LNK2019: unresolved external symbol "int __cdecl flow::OutputFlowlinesNumSteps(class flow::Advection const *,char const *,unsigned __int64,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,bool)" (?OutputFlowlinesNumSteps@flow@@YAHPEBVAdvection@1@PEBD_KAEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@_N@Z) referenced in function "private: int __cdecl VAPoR::FlowRenderer::_outputFlowLines(void)" (?_outputFlowLines@FlowRenderer@VAPoR@@AEAAHXZ)

25>FlowRenderer.obj : error LNK2019: unresolved external symbol "int __cdecl flow::OutputFlowlinesMaxTime(class flow::Advection const *,char const *,double,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,bool)" (?OutputFlowlinesMaxTime@flow@@YAHPEBVAdvection@1@PEBDNAEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@_N@Z) referenced in function "private: int __cdecl VAPoR::FlowRenderer::_outputFlowLines(void)" (?_outputFlowLines@FlowRenderer@VAPoR@@AEAAHXZ)

25>FlowRenderer.obj : error LNK2019: unresolved external symbol "int __cdecl flow::InputSeedsCSV(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,class flow::Advection *)" (?InputSeedsCSV@flow@@YAHAEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@PEAVAdvection@1@@Z) referenced in function "private: virtual int __cdecl VAPoR::FlowRenderer::_paintGL(bool)" (?_paintGL@FlowRenderer@VAPoR@@EEAAH_N@Z)

25>C:\Users\pearse\VAPOR\build\bin\Release\render.dll : fatal error LNK1120: 3 unresolved externals

25>Done building project "render.vcxproj" -- FAILED.

--- a/include/vapor/AdvectionIO.h
+++ b/include/vapor/AdvectionIO.h
@@ -17,24 +17,24 @@ namespace flow
     // Output a certain number of steps from an advection.
     // When `append == false`, a header will also be output.
     // Otherwise, only trajectories are output.
-    auto OutputFlowlinesNumSteps( const Advection*   adv,
-                                  const char*        filename,
-                                  size_t             numStep,
-                                  const std::string& proj4string,
-                                  bool               append ) -> int;
+    FLOW_API auto OutputFlowlinesNumSteps( const Advection*   adv,
+                                           const char*        filename,
+                                           size_t             numStep,
+                                           const std::string& proj4string,
+                                           bool               append ) -> int;
 
     // Output trajectory to a maximum time.
     // When `append == false`, a header will also be output.
     // Otherwise, only trajectories are output.
-    auto OutputFlowlinesMaxTime( const Advection*    adv,
-                                 const char*         filename,
-                                 double              maxTime,
-                                 const std::string&  proj4string,
-                                 bool                append ) -> int;
+    FLOW_API auto OutputFlowlinesMaxTime( const Advection*    adv,
+                                          const char*         filename,
+                                          double              maxTime,
+                                          const std::string&  proj4string,
+                                          bool                append ) -> int;
 
     // Input a list of seeds from lines of CSVs.
-    auto InputSeedsCSV( const std::string& filename, 
-                        Advection*         adv ) -> int;
+    FLOW_API auto InputSeedsCSV( const std::string& filename, 
+                                 Advection*         adv ) -> int;
 
 };
 #endif

